src/Semantic/Util.hs

```diff
@@ -86,6 +93,9 @@ parseFile, parseFileQuiet :: Parser term -> FilePath -> IO term
 parseFile      parser = runTask'     . (parse parser <=< readBlob . fileForPath)
 parseFileQuiet parser = runTaskQuiet . (parse parser <=< readBlob . fileForPath)
 
+fileForPath :: FilePath -> File Language.Language
```

Comment:
This is now internal API used only for REPL-level experiments, which is good enough for me.